src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -391,6 +401,75 @@ protected virtual void Initialize()
             }
         }
 
+        // Scan the extensions directory and Load custom extension.
+        // $$$ - need to finalize this policy. Currently enabled via an AppSettting. 
+        private void LoadCustomExtensions(JObject hostConfig)
+        {
+            var bindingRoot = ConfigurationManager.AppSettings["BYOB_Path"];
+            if (!string.IsNullOrWhiteSpace(bindingRoot))
+            {
+
+                var config = this.ScriptConfig.HostConfig;
+
+                foreach (var dir in Directory.EnumerateDirectories(bindingRoot))
```

Comment:
So the plan is for each different 3rd party extension to have its own directory?